apps/explorer/src/components/transaction-card/TxCardUtils.tsx

```diff
@@ -71,22 +62,26 @@ export function SuiAmount({
     return <span className={styles.suiamount}>--</span>;
 }
 
-export function TxAddresses({ content }: { content: LinkObj[] }) {
+function TxTableHeader({ label }: { label: string }) {
+    return <div className="pl-3">{label}</div>;
+}
+
+function TxTableCol({
+    isHighlighted,
+    children,
+}: {
+    isHighlighted?: boolean;
+    children: ReactNode;
+}) {
     return (
-        <section className={styles.addresses}>
-            {content.map((itm, idx) => (
-                <Fragment key={idx + itm.url}>
-                    {itm.category === 'address' ? (
-                        <AddressLink address={itm.url} />
-                    ) : itm.category === 'object' ? (
-                        <ObjectLink objectId={itm.url} />
-                    ) : (
-                        <TransactionLink digest={itm.url} />
-                    )}
-                    {idx !== content.length - 1 && <ContentArrowRight />}
-                </Fragment>
-            ))}
-        </section>
+        <div
+            className={clsx(
+                'flex h-full items-center rounded px-3',
+                isHighlighted && 'hover:bg-sui-light'
```

Comment:
do we need the `isHighlighted` prop here? seems like we could just use the `hover:bg-sui-light` class ?

Reply:
I am using this for all the columns, not all of them need to be hilighted when hovered

![col-hilite](https://user-images.githubusercontent.com/127577476/225744113-9ccde7cd-ccc3-45f6-a3ce-1618caa039c6.gif)

Reply:
ah I see, missed that.

Reply:
Maybe I should change the prop name to `isHighlightedOnHover` to make it more explicit. Updated the prop name